admin-console/src/main/webapp/src/pages/ApplicationPage.tsx

```diff
@@ -174,6 +170,8 @@ export const ApplicationPage: React.FunctionComponent = () => {
                             ))}
                         </Tr>
                     </Thead>
+                    <Tbody>{ (eventTypesQuery.payload.value.length === 0 ?
+                        'There are no event types found for this application' : '') }</Tbody>
```

Comment:
This is good for now. We could use this component in the future: https://patternfly-react.surge.sh/components/empty-state

Reply:
oh! I will keep this in mind for future.